app/index/attendances/controller.js

```diff
@@ -14,37 +15,102 @@ import AttendanceValidator from "timed/validations/attendance";
  * @extends Ember.Controller
  * @public
  */
-export default Controller.extend({
-  AttendanceValidator,
+export default class AttendanceController extends Controller {
+  @service notify;
+  @service tracking;
+
+  AttendacenValidator = AttendanceValidator;
 
   /**
    * All attendances currently in the store
    *
    * @property {Attendance[]} _allAttendances
    * @private
    */
-  _allAttendances: computed("store", function () {
+  get _allAttendances() {
     return this.store.peekAll("attendance");
-  }),
+  }
 
   /**
    * The attendances filtered by the selected day
    *
    * @property {Attendance[]} attendances
    * @public
    */
-  attendances: computed(
-    "_allAttendances.@each.{date,isDeleted,user}",
-    "model",
-    "user.id",
-    function () {
-      return this._allAttendances.filter((a) => {
-        return (
-          a.get("date").isSame(this.model, "day") &&
-          a.get("user.id") === this.get("user.id") &&
-          !a.get("isDeleted")
-        );
+  get attendances() {
+    return this._allAttendances.filter((a) => {
+      return (
+        a.get("date").isSame(this.model, "day") &&
+        a.get("user.id") === this.user.id &&
+        !a.get("isDeleted")
+      );
+    });
+  }
+
+  /**
+   * Save an attendance
+   *
+   * @method saveAttendance
+   * @param {Changeset} attendance The attendance to save
+   * @public
+   */
+  @action
+  async saveAttendance(attendance) {
+    try {
+      await attendance.save();
+
+      this.notify.success("Attendance was saved");
+    } catch (e) {
+      /* istanbul ignore next */
+      this.notify.error("Error while saving the attendance");
+    }
+  }
+
+  /**
+   * Delete an attendance
+   *
+   * @method deleteAttendance
+   * @param {Attendance} attendance The attendance to delete
+   * @public
+   */
+  @action
+  async deleteAttendance(attendance) {
+    try {
+      await this.store.peekRecord("attendance", attendance.id).destroyRecord();
```

Comment:
I think there is a `@service store` missing

Reply:
done